user/tests/accept/init_env

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+if [ $PARTICLE_TEST_ENV ]; then
+  return
+fi
+
+warn() {
+  >&2 echo "Warning: $1"
+}
+
+error() {
+  >&2 echo "Error: $1"
+  exit 1
+}
+
+require_tool() {
+  which $1 > /dev/null || error "'$1' is not installed"
+}
+
+require_var() {
+  [ ${!1+x} ] || error "$1 is not defined"
+}
+
+export -f warn error require_tool require_var
+
+require_var PLATFORM
+
+# USB vendor/product ID
+if [ ! $USB_VENDOR_ID ] || [ ! $USB_PRODUCT_ID ]; then
+  case $PLATFORM in
+    core)
+      USB_VENDOR_ID=1d50
+      USB_PRODUCT_ID=607d
+      ;;
+    photon)
+      USB_VENDOR_ID=2b04
+      USB_PRODUCT_ID=c006
+      ;;
+    p1)
+      USB_VENDOR_ID=2b04
+      USB_PRODUCT_ID=d008
+      ;;
+    electron)
+      USB_VENDOR_ID=2b04
+      USB_PRODUCT_ID=d00a
```

Comment:
Thanks! Honestly, I don't remember from where I copied these IDs. Since you're working with this branch already, would you mind committing correct IDs?